### PR TITLE
Added dark funky theme

### PR DESCRIPTION
**Pull request by harshilsharma63**

### CODE REVIEW

File: src/themes/darkFunky.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export default {
+    name: 'Dark Funky',
+    thumbnailUrl: 'darkFunky.png',
+    type: 'dark',
+    theme: {{"sidebarBg":"#055374","sidebarText":"#ffffff","sidebarUnreadText":"#ffffff","sidebarTextHoverBg":"#302e30","sidebarTextActiveBorder":"#196caf","sidebarTextActiveColor":"#ffffff","sidebarHeaderBg":"#1f1f1f","sidebarHeaderTextColor":"#ffffff","onlineIndicator":"#399fff","awayIndicator":"#c1b966","dndIndicator":"#e81023","mentionBj":"#0177e7","mentionColor":"#ffffff","centerChannelBg":"#242424","centerChannelColor":"#dddddd","newMessageSeparator":"#e6a419","linkColor":"#e0fd36","buttonBg":"#e80a89","buttonColor":"#000000","errorTextColor":"#ea1410","mentionHighlightBg":"#8b35bd","mentionHighlightLink":"#2bb99a","codeTheme":"monokai","mentionBg":"#0177e7"}},
```
Comment (avasconcelos114) [inline src/themes/darkFunky.js]: You have one too many curly braces on your themes, and those are still using double quotes

Running `npm run check` and `npm run fix` should help you get things sorted here

**Comment by avasconcelos114** [comment]
Thank you for submitting a PR! :tada:

We just realized that we forgot to add in that last step of editing the `src/themes/index.js` file so that the theme is rendered to the page, we'll be sure to add that step to our `CONTRIBUTING.md`

File: src/themes/darkFunky.js (hunk 1 above)
Comment (harshilsharma63) [reply src/themes/darkFunky.js]: Fixed!

**Comment by harshilsharma63** [comment]
@avasconcelos114 done!

**Pull request merged**